waku/waku_test.go

```diff
@@ -1532,7 +1582,9 @@ func TestRequestSentEventWithExpiry(t *testing.T) {
 	w := New(nil, nil)
 	p := p2p.NewPeer(enode.ID{1}, "1", []p2p.Cap{{"shh", 6}})
 	rw := discardPipe()
-	defer rw.Close()
+	defer func() {
+		handleError(t, rw.Close())
+	}()
```

Comment:
[Error handling is good](https://github.com/status-im/status-go/pull/1957/files#r417328424)